src/EditorFeatures/CSharp/FixInterpolatedVerbatimString/FixInterpolatedVerbatimStringCommandHandler.cs

```diff
@@ -35,6 +35,20 @@ public FixInterpolatedVerbatimStringCommandHandler()
         public string DisplayName => CSharpEditorResources.Fix_interpolated_verbatim_string;
 
         public void ExecuteCommand(TypeCharCommandArgs args, Action nextCommandHandler, CommandExecutionContext executionContext)
+        {
+            try
```

Comment:
📝 The call to `nextCommandHandler` itself should _not_ be inside the `try` block